vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass

```diff
@@ -0,0 +1,28 @@
+// Responsive Grid
+// ========================================
+
+// Responsive grid classes from $breakpoints
```

Comment:
Is there a version of the grid that isn't within a breakpoint? For example, does `.grid-1-2` exist if I want 50% all of the time?